documentation/modules/mtv-aio-buffer.adoc

```diff
@@ -0,0 +1,261 @@
+----
++
+* Check for the following lines in the output:
++
+[source,termina]
+----
+...
+\- name: VIRT\_V2V\_EXTRA\_ARGS
+\- name: VIRT\_V2V\_EXTRA\_CONF\_CONFIG\_MAP
+...
+----
+
+.Editing ForkliftController Configuration
+
+* In the `openshift-mtv namespace`, edit the `ForkliftController` object to include the AIO buffer values.
++
+Add the following under the spec section:
++
+[source,termina]
+----
+virt_v2v_extra_args: --vddk-config /mnt/extra-v2v-conf/input.conf
+virt_v2v_extra_conf_config_map: "perf"
+----
+
+
+.Creating a ConfigMap mamed perf
+
+* Create the required ConfigMap using the following command:
++
+[source,termina]
+----
+oc -n openshift-mtv create cm perf`
+----
+
+.Preparing the ConfigMap content
+
+* Convert the desired buffer configuration values to Base64. For example, for 16/4:
++
+[source,termina]
+----
+echo -e "`VixDiskLib.nfcAio.Session.BufSizeIn64KB=16\nvixDiskLib.nfcAio.Session.BufCount=4`" | base64`
+----
+
++
+The output will similar to this:
++
+[source,termina]
+----
+Vml4RGlza0xpYi5uZmNBaW8uU2Vzc2lvbi5CdWZTaXplSW42NEtCPTE2CnZpeERpc2tMaWIubmZjQWlvLlNlc3Npb24uQnVmQ291bnQ9NAo=
+----
+
+.Editing the ConfigMap
+
+* Update the perf ConfigMap with the Base64 string under the binaryData section, for example:
++
+[source,termina]
+----
+apiVersion: v1
+kind: ConfigMap
+binaryData:
+  input.conf: Vml4RGlza0xpYi5uZmNBaW8uU2Vzc2lvbi5CdWZTaXplSW42NEtCPTE2CnZpeERpc2tMaWIubmZjQWlvLlNlc3Npb24uQnVmQ291bnQ9NAo=
+metadata:
+  name: perf
+  namespace: openshift-mtv
+----
+
+.Restarting the Forklift Controller Pod
+
+* Restart the forklift-controller pod to apply the new configuration.
+
+* Ensure the `VIRT_V2V_EXTRA_ARGS` environment variable reflects the updated settings.
+
+.Verifying migration logs
+
+* Run a migration plan and check the logs of the migration pod. Confirm that the AIO buffer settings are passed as parameters, particularly the `--vddk-config value`.
++
+For example:
++
+[source,termina]
+----
+exec: /usr/bin/virt-v2v … --vddk-config /mnt/extra-v2v-conf/input.conf
+----
+
++
+Sample log excerpt:
++
+[source,termina]
+----
+Buffer size calc for 16 value:
+(16 * 64 * 1024 = 1048576)
+nbdkit: vddk[1]: debug: [NFC VERBOSE] NfcAio_OpenSession:
+Opening an AIO session.
+nbdkit: vddk[1]: debug: [NFC INFO] NfcAioInitSession:
+Disabling
+read-ahead buffer since the AIO buffer size of 1048576 is >=
+the read-ahead buffer size of 65536. Explicitly setting flag
+'`NFC_AIO_SESSION_NO_NET_READ_AHEAD`'
+nbdkit: vddk[1]: debug: [NFC VERBOSE] NfcAioInitSession: AIO Buffer Size is 1048576
+nbdkit: vddk[1]: debug: [NFC VERBOSE] NfcAioInitSession: AIO Buffer
+Count is 4
+----
++
+[NOTE]
+====
+The above logs were when using `debug_level = 4`
+====
+
+.Inspecting ConfigMap values Content are in the Migration Pod
+
+* Log in to the migration pod and verify the buffer settings using the following command:
++
+[source,termina]
+----
+`cat /mnt/extra-v2v-conf/input.conf`
```

Comment:
replace  this line:  `cat /mnt/extra-v2v-conf/input.conf`
with this one:  cat /mnt/extra-v2v-conf/input.conf